plugins/modules/panos_sag.py

```diff
@@ -84,6 +84,7 @@
         type: list
         elements: str
         required: false
+        default: []
```

Comment:
I'm not entirely sure this is correct....?  Did you verify that a list defaults to an empty list if it's not specified?  I would have expected Python `None` here.